### PR TITLE
Fix typos in docs and deprecation: zcml_package_metas and load_zcml.

### PR DESCRIPTION
**Pull request by mauritsvanrees**
- Fix typo in docs: it is not `zcml_package_meta` singular, but `zcml_package_metas` plural. I tried the singular, but it was not working, so had to dive into the code.

- Fix typo in deprecation warning: there was never a `zcml` dict setting, only `load_zcml`. I kept seeing a deprecation warning that I should not be using a `zcml` dict, but I was not actually using it, so I never knew what or where to fix. And the setting is not actually removed, but it is deprecated.  Changed the warning accordingly.

### CODE REVIEW

File: hooks/pre_gen_project.py
Comment (stevepiercy) [inline hooks/pre_gen_project.py:35]: I think this is more explicit and helpful.

```suggestion
        "The 'load_zcml' dict setting was deprecated in version 2.0. Instead use the name of its keys prefixed with 'zcml_' as the new variable names."
```

Also I don't know if we should update the change log for 2.0.0 to mention that the initial removal was restored in 2.0.1, or clarify the entry for 2.0.1. I missed that on my first reading, and it wasn't really clear at first.

Maybe this?

```
- Fix: Implement a fallback to use ``load_zcml`` dict configuration, effectively making the breaking change of its removal in 2.0.0 a deprecation instead. Fixes #18.
  [@ericof, 2024-03-01]
```

**Review by stevepiercy — CHANGES_REQUESTED** [review]

**Review by stevepiercy — APPROVED** [review]

File: hooks/pre_gen_project.py
Comment (mauritsvanrees) [reply hooks/pre_gen_project.py]: I have copied your other suggestion to ``CHANGES.rst`.

**Comment by stevepiercy** [comment]
Approved again! OK by me to merge, unless you want to wait for @jensens to review, too.

**Pull request merged**
